main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	_ "embed" // this is required in order for go:embed to work
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/launchdarkly/sdk-test-harness/framework"
+	"github.com/launchdarkly/sdk-test-harness/framework/harness"
+	"github.com/launchdarkly/sdk-test-harness/framework/ldtest"
+	"github.com/launchdarkly/sdk-test-harness/sdktests"
+	"github.com/launchdarkly/sdk-test-harness/servicedef"
+)
+
+const defaultPort = 8111
+const statusQueryTimeout = time.Second * 10
+
+//go:embed VERSION
+var versionString string // comes from the VERSION file which we update for each release
```

Comment:
That's an interesting feature but I'm not sure how reliable the module version would be in this context. The application binary is (I think) getting built before a new version tag has been created, so it would probably be picking up the _previous_ module version.